src/plugins/UI-plugin.js

```diff
@@ -0,0 +1,26 @@
+import { EventEmitter } from "events";
```

Comment:
Forgive me that linter is not configured, but all quotes in the project are single.)